drivers/mssql/mssql.go

```diff
@@ -459,6 +466,68 @@ ORDER BY i.index_id
 	return nil
 }
 
+const query = `select schema_name(obj.schema_id) as schema_name,
```

Comment:
If you have the extra power, it would be great if you could change the reserved words in the SQL to uppercase.

Reply:
fixed in [e47d996](https://github.com/k1LoW/tbls/pull/341/commits/e47d996d6407e7ef66f61af8b8468cba327f921f), [faa445f](https://github.com/k1LoW/tbls/pull/341/commits/faa445f5a7c00445a23cf3855ad164a52f8ae750) and [588ced1](https://github.com/k1LoW/tbls/pull/341/commits/588ced15fb2a3a1f817eed1bfb2f7fb15dfca45e)